drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -980,6 +992,18 @@ int qspi_wait_while_rpu_awake(const struct device *dev)
 		k_msleep(1);
 	}
 
+	/* Configure DTS settings */
+	if (val & RPU_AWAKE_BIT) {
+		nrf_qspi_phy_conf_t qspi_phy_dts;
```

Comment:
This may leave some fields not initialized, i.e. having some random values from stack (this is the case with `dpmen` currently). It's safer to use:
```suggestion
		nrf_qspi_phy_conf_t qspi_phy_dts = {
			...
```